### ISSUE TITLE
Glove pain

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11226]
**HPO term**

**Synonym to be added**
Glove pain
Glove type pain

**broad/exact/narrow/plural/abbreviation ?**

 I am not sure about adding glove pain or glove-like pain to the term Hand pain HP:0046505 or adding it as a Spatial Pattern type HP:0012836

PMID: 33320513 Peripheral neuropathy, a common neurologic problem encountered by family physicians, can be classified clinically by the anatomic pattern of presenting symptoms and, if indicated, by results of electrodiagnostic studies for axonal and demyelinating disease. The prevalence of peripheral neuropathy in the general population ranges from 1% to 7%, with higher rates among those older than 50 years. Common identifiable causes include diabetes mellitus, nerve compression or injury, alcohol use, toxin exposure, hereditary diseases, and nutritional deficiencies. Peripheral neuropathy is idiopathic in 25% to 46% of cases.
arly peripheral neuropathy may present as sensory alterations that are often progressive, including sensory loss, numbness, pain, or burning sensations in a “**stocking and glove**” distribution of the extremities.

PMID: 36786097  Hyperglycemia and dyslipidemia are the primary causes of DPN (Painful diabetic peripheral neuropathy) in patients with type 1 and 2 diabetes, respectively. The pattern of pain from DPN is described as “glove and stocking”.
DPN-induced pain is described as burning, electric, sharp, and dull aching with various pain intensities.

**Comment by pnrobinson** [comment]
@carolina-f  The findings in peripheral neuropathy include more than pain, and so I think that a modifier called "Stocking and glove distribution" is the best.

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001335
- label: Stocking-glove distribution
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Distribution of findings in hands and feet with a distal to proximal gradient of severity (being more severe in the more distal locations).
- comment: Early peripheral neuropathy may present as sensory alterations that are often progressive, including sensory loss, numbness, pain, or burning sensations in a stocking and glove distribution of the extremities.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11226
- synonyms:
